packages/dd-trace/src/plugins/plugin.js

```diff
@@ -5,6 +5,8 @@
 const dc = require('diagnostics_channel')
 const { storage } = require('../../../datadog-core')
 
+const storesByContext = new WeakMap()
```

Comment:
What happens if multiple subscriptions exist for the same event? Why not just use an instance property?